src/App.tsx

```diff
@@ -1,12 +1,47 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
+import { getAll, get5First, getRed } from './api/goods';
+import { GoodsList } from './GoodsList';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
 
-const App: React.FC = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+  const getGoods = async (inputGoods: () => Promise<Good[]>) => {
+    const response = await inputGoods();
+
+    setGoods(response);
```

Comment:
it's already data, not response from the server

```suggestion
    const goods = await inputGoods();

    setGoods(goods);
```